app/containers/Wallet/Weeks.jsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  TEXT_WARNING_LIGHT,
+  TEXT_SECONDARY,
+  TEXT_SUCCESS,
+} from 'style-constants';
+
+import { getFormattedNum3 } from 'utils/numbers';
+import { getFormattedDate } from 'utils/datetime';
+import { FULL_MONTH_NAME_DAY_YEAR } from 'utils/constants';
+
+import calendarImage from 'images/calendar.svg?inline';
+import currencyPeerImage from 'images/currencyPeer.svg?inline';
+import rewardIcon from 'images/add.svg?external';
+
+import P from 'components/P';
+import Icon from 'components/Icon';
+import IconStyled, { IconHover } from 'components/Icon/IconStyled';
+import Span from 'components/Span';
+import Base from 'components/Base';
+import BaseRounded from 'components/Base/BaseRounded';
+import SmallImage from 'components/Img/SmallImage';
+
+import messages from './messages';
+
+const data = [
+  {
+    id: 5,
+  },
+  {
+    id: 4,
+  },
+  {
+    id: 3,
+  },
+  {
+    id: 2,
+  },
+  {
+    id: 1,
+  },
+];
+
+const BaseRoundedLi = BaseRounded.extend`
+  position: relative;
+  display: flex;
+  align-items-cemter;
+  justify-content: space-between;
+
+  ${IconStyled} {
+    ${IconHover({ color: TEXT_SUCCESS })};
+  }
+`.withComponent('li');
+
+const WeekNumber = ({ number, locale }) => (
+  <P>
+    <Span className="mr-3" fontSize="24" bold>
+      Week {number}
+    </Span>
+
+    <Span>
+      {getFormattedDate(
+        1565692922 + 604800 * number,
+        locale,
+        FULL_MONTH_NAME_DAY_YEAR,
+      )}
+      {' — '}
+      {getFormattedDate(
+        1565692922 + 604800 * number + 604800,
```

Comment:
Move 1565692922 to .env file
This will change depending on environment (test, prod, local)
604800 needs to be in constants. it will be easier to read code and reusable (`seconds_in_week`)